src/Bridges/bridgeoptimizer.jl

```diff
@@ -222,37 +222,72 @@ function MOI.set(b::AbstractBridgeOptimizer,
     return MOI.set(b.model, attr, indices, values)
 end
 
+# function MOI.get(::MOI.ModelLike, attr::MOI.AbstractConstraintAttribute,
+#     bridge::AbstractBridge)
+# throw(ArgumentError("Constraint bridge of type `$(typeof(bridge))` does not support accessing the attribute `$attr`."))
+# end
+
 # Constraint attributes
 function MOI.get(b::AbstractBridgeOptimizer,
                  attr::MOI.AbstractConstraintAttribute,
                  ci::CI)
     if is_bridged(b, typeof(ci))
-        if MOI.is_set_by_optimize(attr)
-            MOI.get(b, attr, bridge(b, ci))
-        else
+        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
```

Comment:
Indeed, ConstraintFunction and ConstraintSet currently need to be obtained from the `bridged` model (this is what will change with light bridges, then the bridges will have to implement these getters).

Reply:
Ok so I leave this condition.